prov/rxm/src/rxm_conn.c

```diff
@@ -600,14 +600,18 @@ int rxm_msg_eq_progress(struct rxm_ep *rxm_ep)
 	}
 
 	while (1) {
+		memset(entry, 0, RXM_MSG_EQ_ENTRY_SZ);
```

Comment:
I'm not sure the memset is necessary. What problem does this solve? 

As the entry is passed to rxm_eq_read, the function should overwrite any of the existing values on success, and leave them uninitialized if a failure occurs. `entry->rd` is set via the return value, so that doesn't have to be initialized either. 

At this point, I think the memset only makes the contents of the stack memory clean, at the expense of several CPU instructions -- without a tangible benefit. Am I missing something?

Reply:
@jswaro 
No, you are not. I just tried to align it with rxm_conn_progress, in order to memory clean.

Reply:
I'd prefer to not have the `memset` done here. If you feel that it is necessary, perhaps you can move it directly after the `alloca()` function call to clear the content of memory on allocation, but I do think that it is unnecessary.

Reply:
@jswaro done.